js/build-env-config.js

```diff
@@ -12,14 +12,10 @@ if (environment !== `PRODUCTION`) {
   habitat.load(`config/default.env`);
 }
 
-const HOST = habitat.get(`HOST`);
 const PORT = habitat.get(`PORT`);
 const LEARN_MORE_LINK = habitat.get(`LEARN_MORE_LINK`);
-const ORIGIN = HOST + (PORT ? `:${PORT}` : ``);
```

Comment:
`ORIGIN` not being used anywhere now. We don't need it... and hence no need to have `HOST`.